DrawerKit/DrawerKit/Public API/DrawerDisplayController.swift

```diff
@@ -13,7 +13,7 @@ public final class DrawerDisplayController: NSObject {
     public let configuration: DrawerConfiguration
 
     weak var presentingVC: UIViewController?
-    /* strong */ var presentedVC: (UIViewController & DrawerPresentable)
+    weak var presentedVC: (UIViewController & DrawerPresentable)!
```

Comment:
This needs to be optional and should be unwrapped when used the same way we are doing with `presentingVC`.